site/en/integrations/integrate_with_sentencetransformers.md

```diff
@@ -6,222 +6,176 @@ title: Movie Search Using Milvus and SentenceTransformers
-
-We are going to use `gdown` to grab the zip from Google Drive and then decompress it with the built-in `zipfile` library.
-
 ```python
-import gdown
-url = 'https://drive.google.com/uc?id=11ISS45aO2ubNCGaC3Lvd3D7NT8Y7MeO8'
-output = './movies.zip'
-gdown.download(url, output)
-
-import zipfile
-
-with zipfile.ZipFile("./movies.zip","r") as zip_ref:
-    zip_ref.extractall("./movies")
+from datasets import load_dataset
+from pymilvus import MilvusClient, connections
+from pymilvus import FieldSchema, CollectionSchema, DataType, Collection
+from sentence_transformers import SentenceTransformer
+from tqdm import tqdm
 ```
 
-## Global parameters
+We'll define some global parameters,
+```python
+embedding_dim = 384
+collection_name = "movie_embeddings"
+```
 
-Here we can find the main arguments that need to be modified for running with your own accounts. Beside each is a description of what it is.
+## Downloading and Opening the Dataset
+In a single line, `datasets` allows us to download and open a dataset. The library will cache the dataset locally and use that copy next time it is run. Each row contains the details of a movie that has an accompanying Wikipedia article. We only make use of the `Title` and `PlotSummary` columns.
 
 ```python
-# Milvus Setup Arguments
-COLLECTION_NAME = 'movies_db'  # Collection name
-DIMENSION = 384  # Embeddings size
-COUNT = 1000  # Number of vectors to insert
-MILVUS_HOST = 'localhost'
-MILVUS_PORT = '19530'
-
-# Inference Arguments
-BATCH_SIZE = 128
-
-# Search Arguments
-TOP_K = 3
+ds = load_dataset("vishnupriyavr/wiki-movie-plots-with-summaries", split="train")
+print(ds)
 ```
 
-## Setting up Milvus
-
+## Connecting to the Database
 At this point, we are going to begin setting up Milvus. The steps are as follows:
 
-1. Connect to the Milvus instance using the provided URI.
-
-    ```python
-    from pymilvus import connections
-
-    # Connect to Milvus Database
-    connections.connect(host=MILVUS_HOST, port=MILVUS_PORT)
-    ```
-
-2. If the collection already exists, drop it.
+1. Create a Milvus Lite database in a local file. (Replace this URI to the server address for Milvus Standalone and Milvus Distributed.)
 
-    ```python
-    from pymilvus import utility
-
-    # Remove any previous collections with the same name
-    if utility.has_collection(COLLECTION_NAME):
-        utility.drop_collection(COLLECTION_NAME)
-    ```
+```python
+connections.connect(uri="./sentence_transformers_example.db")
+```
 
-3. Create the collection that holds the id, title of the movie, and the embeddings of the plot text.
+2. Create the data schema. This specifies the fields that comprise an element including the dimension of the vector embedding.
 
-    ```python
-    from pymilvus import FieldSchema, CollectionSchema, DataType, Collection
+```python
+fields = [
+    FieldSchema(name='id', dtype=DataType.INT64, is_primary=True, auto_id=True),
+    FieldSchema(name='title', dtype=DataType.VARCHAR, max_length=256),
```

Comment:
Nit, is there another field from the data source that we can use to demo metadata filtering (say timestamp, author etc)?

Reply:
Added film origin and release year